src/services/cleeng.service.ts

```diff
@@ -1,12 +1,16 @@
+import cleengAuthService from '#src/services/cleeng.auth.service';
+
 export const getBaseUrl = (sandbox: boolean) => (sandbox ? 'https://mediastore-sandbox.cleeng.com' : 'https://mediastore.cleeng.com');
 
-export const performRequest = async (sandbox: boolean, path: string = '/', method = 'GET', body?: string, jwt?: string) => {
+export const performRequest = async (sandbox: boolean, path: string = '/', method = 'GET', body?: string, authenticate = false) => {
```

Comment:
Maybe nice to use a named option instead of the plain boolean for readability when called.
```suggestion
export const performRequest = async (sandbox: boolean, path: string = '/', method = 'GET', body?: string, options?: {authenticate?: boolean} = {authenticate: true}) => {
```

Reply:
Yes, good call! I almost also added `sandbox` to the options... 😅

Reply:
Actually that would not be a terrible thing to do next time we touch this code